File: problems/problems_3145/solution.go
package problem3145

import (
	"encoding/json"
	"log"
	"strings"
)

func findProductsOfElements(queries [][]int64) []int {

}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var queries [][]int64

	if err := json.Unmarshal([]byte(inputValues[0]), &queries); err != nil {
		log.Fatal(err)
	}

	return findProductsOfElements(queries)
}

File: problems/problems_3145/solution.ts
function findProductsOfElements(queries: number[][]): number[] {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const queries: number[][] = JSON.parse(inputValues[0]);
	return findProductsOfElements(queries);
}
